test/suite/test_backup01.py

```diff
@@ -187,7 +188,24 @@ def test_checkpoint_delete(self):
         self.assertRaisesWithMessage(wiredtiger.WiredTigerError,
             lambda: self.session.checkpoint("name=three,drop=(two)"), msg)
         self.session.checkpoint()
+
+        # Confirm that a named checkpoint created after a backup cursor can be dropped.
+        # Need to pause a couple seconds as checkpoints created immediately after a backup
```

Comment:
I might add a little more the the "immediately after" comment to more clearly indicate "within the same second" because future us/me may not leap to that. Same in any other tests that might have a similar comment.